### PR TITLE
A kruglov

### PR DESCRIPTION
**Pull request by Darshniderd**
Partially completed the third task "Game"

### CODE REVIEW

File: Practice/kruglov/home_task_2/Third_task.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class Human:
```
Comment (IlyaOrlov) [inline Practice/kruglov/home_task_2/Third_task.py]: Great! But what about changing the action properties dynamically (e.g. speeding up or slowing down)? We can use just one class for the game character (let it be class Human), and (this is a tip) several classes for different types of action (run, fly, run quicker, run slower, etc). So class Human may include some field responsible for the action which can be an object of any of classes Run, Fly, QuickRun... (check this for more details: https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D1%80%D0%B0%D1%82%D0%B5%D0%B3%D0%B8%D1%8F_(%D1%88%D0%B0%D0%B1%D0%BB%D0%BE%D0%BD_%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F))

File: Practice/kruglov/home_task_3/PyLec4T2.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+number = list(input("Please enter a five-digit number: "))
```
Comment (IlyaOrlov) [inline Practice/kruglov/home_task_3/PyLec4T2.py]: What is the reason to use list here?

Comment (Darshniderd) [reply Practice/kruglov/home_task_3/PyLec4T2.py]: No reason. Fixed.

File: Practice/kruglov/home_task_3/PyLec4T1.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+for x in range(1, 101):
+    if x % 3 == 0 and x % 5 == 0:
```
[the remark this answers is not in the document]
Comment (Darshniderd) [reply Practice/kruglov/home_task_3/PyLec4T1.py]: Of course you're right!)

**Pull request merged**